pages/dev.js

```diff
@@ -18,21 +18,107 @@ export default function Home() {
           email="amanda.dupell@hpe.com"
           linkedIn="https://www.linkedin.com/in/amanda-dupell-profile/"
         />
-        <Paragraph size="large">
-          You are expecting an exciting internship diving into the open-source
-          world and the community of grommet.io, so far we have more than +3,700
-          members on our slack channel, a great team, loads of enhancements
-          requests and an exciting road-map for both grommet and the HPE
-          Design-System.
+        <Text size="4xl" weight="bold" >Hi, there!</Text>
+        <Paragraph size="large">My name is Amanda Dupell and I am super excited to start
+          working as a UI/UX Developer Intern as a part of HPE's Experience 
+          Studio and Design System team!
         </Paragraph>
-        <Paragraph size="large">
-          Shimrit (Shimi) Yacobi, will be your “HPE Onboarding Buddy” (located
-          in Fort Collins, CO) and a primary contact to get you oriented to HPE
-          which includes getting your Mac set up, getting connected to the HPE
-          Network, and addressing any questions about the team and internship
-          project and such.
+        <Paragraph size="medium">
+          As I begin my internship at HPE, I have a lot of exciting projects and 
+          learning opportunities to look forward to this summer and beyond. One of
+          the first things I will be working on is the continuous improvement of 
+          accessibility across {
+            <Anchor href="https://github.com/grommet/grommet">Grommet</Anchor>
```

Comment:
This Anchor might be throwing an axe warning due to a missing a11yTitle (aria-label)